docs/standards/smirnoff.md

```diff
@@ -521,14 +528,48 @@ For example:
 
 Currently, only `potential="k*(1+cos(periodicity*theta-phase))"` is supported, where we utilize the functional form:
 ```
-U = \sum_{i=1}^N k_i * (1 + cos(periodicity_i * phi - phase_i))
+U = \sum_{i=1}^N k_i/idivf_i * (1 + cos(periodicity_i * phi - phase_i))
```

Comment:
Nice catch! 

(not blocking) I might also suggest this one while we're here (this appears to be a copy-paste error from the impropers section below, but in that section we clarify that we're talking about CHARMM, whereas here we just mysteriously mention phi with no warning)

```suggestion
U = \sum_{i=1}^N k_i/idivf_i * (1 + cos(periodicity_i * theta - phase_i))
```

Reply:
Ah great catch there, yes this would be a good fix -- thank you!